template/AssetName.bat

```diff
@@ -1,2 +1,13 @@
+:: :: Following is the commands that needs to be called in the bat
```

Comment:
Aah, ok. Good thinking, but I don't think it gets the point across.

The way it's managed in Python docstrings is to indent children of a section, and I think that could work here as well..

```bash
:: Following is the commands that needs to be called in the bat
:: always call _mkproject
::
:: Variables:
::   %~n0 is the name of this project as the name of the .bat
::   %~dp0 is drive plus path variable
```

Btw, what does `always call _mkproject` mean? Try and think of it as someone who reads it for the first time. I'm not sure it makes sense. Should he call _mkproject? When? Always? Like once a day, once per shot?